include/rdma/fabric.h

```diff
@@ -754,6 +754,28 @@ struct fi_param {
 int fi_getparams(struct fi_param **params, int *count);
 void fi_freeparams(struct fi_param *params);
 
+enum fi_log_subsys {
+	FI_LOG_CORE,
+	FI_LOG_FABRIC,
+	FI_LOG_DOMAIN,
+	FI_LOG_EP_CTRL,
+	FI_LOG_EP_DATA,
+	FI_LOG_AV,
+	FI_LOG_CQ,
+	FI_LOG_EQ,
+	FI_LOG_MR,
+	FI_LOG_CNTR,
+	FI_LOG_SUBSYS_MAX
+};
+
+enum fi_log_level {
+	FI_LOG_WARN,
+	FI_LOG_TRACE,
+	FI_LOG_INFO,
+	FI_LOG_DEBUG,
+	FI_LOG_MAX
+};
```

Comment:
We shouldn't need this commit now that we're just going to install the headers

Reply:
Done. Thanks!